.tools/dart-sdk/bin/resources/dartdoc/resources/readme.md

```
@@ -0,0 +1,36 @@
+through the `dart doc` command.
+
+To learn more about generating and viewing the generated documentation,
+check out the [`dart doc` documentation][].
+
+[`dart doc` documentation]: https://dart.dev/tools/dart-doc
+
+## Third-party resources
+
+## highlight.js
+
+Generated from https://highlightjs.org/download/ on 2021-07-13.
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)